python/tvm/contrib/debugger/debug_executor.py

```diff
@@ -93,9 +98,16 @@ class GraphModuleDebug(graph_executor.GraphModule):
     dump_root : str
         To select which folder the outputs should be kept.
         None will make a temp folder in /tmp/tvmdbg<rand_string> and does the dumping
+    iters_num : int
```

Comment:
With `time_evaluator` and `GraphExecutor.benchmark`, this parameter is called `number`. It is not that descriptive, but I think it would be worthwhile to keep the parameter names the same.

Also, you can maybe just copy and paste the description of `number` and `repeat` from `graph_executor.py`?